c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,63 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+class LTagProcessor(BlockProcessor):
```

Comment:
2 blank lines required before a class definition.
To get hints about the coding standard you may run the linter available in the project by typing the following command:
```
make lint
```
This is really useful to check that your syntax is correct.